modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -171,3 +159,16 @@ $ ipl c
 ----
 +
 See link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-ipl[IPL] in {ibm-name} Documentation.
+endif::ibm-z[]
+ifdef::ibm-z-lpar[]
+. Transfer the initramfs, kernel, parameter files, and {op-system} images to the LPAR, for example with FTP. For details about how to transfer the files with FTP and boot, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/interactively_installing_rhel_over_the_network/index#installing-in-an-lpar_booting-the-installation-media[Booting the installation on {ibm-z-name} to install {op-system-base} in an LPAR].
```

Comment:
may not be directly related, but I see that the above snippet points to RHEL 7 docs and this one to RHEL 9. Are we expecting any consistency of versions here?
I do not have a lot of context. Please ignore if the question doesn't make sense.

Reply:
Good catch, when we added LPAR the RHEL docs team had already restructured the installation part. Missed changing the z/VM link.